.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
My comment regarding the python versions where direct at this:

>  the original configuration file did not seem to be testing all the Python versions. 

> https://travis-ci.org/NESII/pyugrid/jobs/87780816#L409. Does that make sense?

You need to check if all the run and build dependencies exists for the python version you are testing against.   Since the current version of pyugrid installs just fine for 3.4 my guess is that you are missing a new dependency introduced by this PR.

> That's a nice example! 

Indeed.

> Are they testing Python 3?

No.  The point of the example is to show how to separate the context of testing (build test /code test), which I am still :-1: BTW.

>  It looks like they are also installing miniconda in a similar way to the in-question 

They do because their tests are very long and downloading the "right" version of miniconda makes it faster.  That only works because they test against 2.7 only (and maybe in the future 2.7 and 3.4).  However, when testing against several python versions, like we do, that advantage is lost.  We still need to create ENVs for all those python versions.   So a simpler travis that downloads only version of minconda is cleaner and better.